Game/Source/GDKTestGyms/Private/Disco387GameStateBase.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "Disco387GameStateBase.h"
+
+#include "SpatialGDKSettings.h"
+
+void ADisco387GameStateBase::PostInitializeComponents()
+{
+	Super::PostInitializeComponents();
+
+	// Modify settings here that are desired for this scenario. We modify them here so that it's possible
+	// to run a single assembly against multiple scenarios. This might not actually be necessary, in which
+	// case we can incorporate these changes into the config file directly.
```

Comment:
If this is hypothetical why do it now?

Reply:
This allows us to run multiple scenarios using the same bundle. It might pan out that we never want/need to do this, but better to keep the option open for now.